UndertaleModTool/CommunityScripts/FindObjectsInRooms.csx

```diff
@@ -16,7 +16,7 @@ List<string> splitStringsList = new List<string>();
 List<string> gameObjectsUsedList = new List<string>();
 string InputtedText = "";
 InputtedText = SimpleTextInput("Menu", "Enter name(s) of game object(s)", InputtedText, true);
-string[] IndividualLineArray = InputtedText.Split(new[] { "\n" }, StringSplitOptions.RemoveEmptyEntries | StringSplitOptions.TrimEntries);
+string[] IndividualLineArray = InputtedText.Split(new[] { '\n', '\v' }, StringSplitOptions.RemoveEmptyEntries | StringSplitOptions.TrimEntries);
```

Comment:
This case should get handled in SimpleTextInput rather than in the script itself.